includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
Have you tried `snmpwalk -v2c -cCOMMUNITY HOSTNAME   -M /opt/librenms/mibs/ -m IF-MIB ifNumber`?